### PR TITLE
Add owner and charter

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: MISSION.md [hunk 1]
```diff
@@ -76,17 +76,22 @@ Meeting recordings will be posted to a YouTube channel for offline viewing.
 
 A public Google Group has been created to facilitate offline discussion outside of Github issues or Pull Requests.
 
-https://groups.google.com/forum/#!forum/open-cluster-management open-cluster-management@googlegroups.com
```
Comment (deads2k) [inline MISSION.md:79]: does this exist?

**Comment by deads2k** [comment]
The idea and intent lgtm.  intentionally not tagging to allow others time to review and to work out name.

File: wg-architecture/charter.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# WG Architecture Charter
```
Comment (deads2k) [inline wg-architecture/charter.md]: if the group is going to be long-standing, people familiar with kube are probably more used to this being called a sig.

File: MISSION.md (hunk 1 above)
Comment (qiujian16) [reply MISSION.md:79]: it should have been created. I will double check that.

File: wg-architecture/charter.md (hunk 2 above)
Comment (qiujian16) [reply wg-architecture/charter.md]: changes to sig

File: MISSION.md (hunk 1 above)
Comment (mdelder) [reply MISSION.md:79]: It does -- I have the privacy settings set right now but will open it up once this document solidifies.

**Review by mdelder — APPROVED** [review]
Looks good to me. 

If we're going to form the "Architecture SIG", then I don't think we would have a separately named Technical Steering Committee.

**Review by cadawson — COMMENTED** [review]
README
Line 10: How are clusters configured, audited, secured, access controlled, and managed for quota or cost?
Line 23: We expect that, over time, the project will contribute to an appropriate foundation for stewardship.
Line 26: The English text of the CNCF Code of Conduct that was current at the time this was written is made available here for convenience, but you should consult the version at the link provided to see the latest content.
Lines 31-37: List formatting
* The use of sexualized language or imagery
* Personal attacks
* Trolling or insulting/derogatory comments
* Public or private harassment
* Publishing others' private information, such as physical or electronic addresses, without explicit permission
* Other unethical or unprofessional conduct.
Line 44: Anyone who is interested in getting involved is welcomed to contribute in a number of ways:
Line 46: Join the recurring meeting forums (see the section that follows for details) to provide input as a stakeholder and help validate proposed use cases.
Lines 46-48: I think this should be a list, which you can format by adding asterisks at the beginning:
* Join the recurring meeting forums (see below) to provide input as a stakeholder and help validate proposed use cases.
* Suggest enhancements via github.com/open-cluster-management/enhancements for consideration to the community.
* Contribute to development via Pull Request for new enhancements or defect fixes.
Line 54: To ensure opportunities for broad user contributions, a public forum will be hosted to demonstrate new capabilities, solicit feedback, and offer a forum for real time Q&A.
Lines 58, 60, 70, 72: Replace "the month" with "each month".
Line 77: Replace "Github" with "GitHub".
Line 86: **Special Interest Groups (SIGs)** are...
charter
Line 18: Cross-cutting and Externally-Facing Processes
Line 29: Ensure the agenda is populated 24 hours in advance of the community meeting, or the community meeting is cancelled. 
Lines 30-32: Add periods to the end of each of the list items. 
Line 32: Uphold the Code of Conduct, especially in terms of personal behavior and responsibility.
Overall: This looks very good! I would recommend using sentence style capitalization for the sections. Only the first word is capitalized, unless a later word is a proper noun. For example: Cross-cutting and externally-facing processes. This is not required, but this is typically what we use, and it might be more consistent

**Comment by cadawson** [comment]
Hi @qiujian16 . I ran through the website files. I submitted some PRs for some small changes. It looked similar to the other content that I looked at in the readme, so you might be able to merge these PRs and just paste that in the other ones. It is up to you. 
The following links were broken for me: 
In the website/content/community/resource.md file: the mailing group link was a 404.
In the same file, the DCO link was broken.
As always, let me know if something is not clear, or if we should make any changes. Thanks!

**Pull request merged**